docs/usage/administration/admin_faq.md

```diff
@@ -71,6 +71,9 @@ output-directory
 │       ├───invite_state
 │       └───knock_state
 └───user_data
+    ├───account_data
+    │   ├───global
+    │   └───<room_id>
```

Comment:
I wonder why not do `rooms` > `account_data` and then just have a `user_data` > `account_data` (or `global_account_data`)?

Reply:
This has no special reasons. My thinking was that account data is user related rather than room related. I will change it.

Reply:
Not anymore. The PR has now been merged.

Reply:
It isn't a big deal, just curious what the thinking was!

They're related to both users & rooms, which is annoying in this case. 😄